### PR TITLE
Adds new Monthly Banner Variant to tipping process

### PR DESCRIPTION
**Pull request by ryanml**
Fixes: https://github.com/brave/brave-browser/issues/5996

Screen Shot 2019-09-13 at 3 13 14 PM

Screen Shot 2019-09-13 at 3 46 39 PM

## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [x] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
Defined in issue

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by ryanml** [comment]
@NejcZdovc 

1. Bottom padding increased
Screen Shot 2019-09-18 at 11 33 22 AM

2. I could not reproduce a scrollbar, but I tried to tighten up padding a little. Truth is this is another element added in the panel and unless we increase the max height of the extension window this will happen, cc: @petemill for thoughts, I think we are fine for this PR though
Screen Shot 2019-09-18 at 11 33 11 AM

3. I could not reproduce this, can you confirm that you have the latest hash of brave-ui installed for this PR?

4. This problem is currently present on release build and all other channels, I will file an issue and fix separately.

5. This issue has been fixed

File: components/brave_rewards/resources/tip/components/app.tsx [hunk 1]
```diff
@@ -28,18 +29,20 @@ export class App extends React.Component<Props, {}> {
     return this.props.actions
   }
 
-  getTipBanner = (url: string, publisher: RewardsTip.Publisher, mediaMetaData: RewardsTip.MediaMetaData | undefined) => {
+  getTipBanner = (url: string, monthly: boolean, publisher: RewardsTip.Publisher, mediaMetaData: RewardsTip.MediaMetaData | undefined) => {
```
Comment (NejcZdovc) [inline components/brave_rewards/resources/tip/components/app.tsx]: maybe instead of passing arg in, we can just access props directly in this func? wdyt?

Comment (ryanml) [reply components/brave_rewards/resources/tip/components/app.tsx]: fine by me, done

File: .storybook/locale.ts [hunk 2]
```diff
@@ -179,6 +180,8 @@ const locale: Record<string, string> = {
   rewardsBackupText3: 'You can always use this key to get your wallet back if something happens to your browser or computer. But make sure to protect it — anyone who gets this key could steal your wallet. It\'s probably safest to write it down on a piece of paper, or wherever else you keep important info.',
   rewardsBannerText1: 'Thanks for stopping by. We joined Brave’s vision of protecting your privacy because we believe that fans like you would support us in our effort to keep the web a clean and safe place to be.',
   rewardsBannerText2: 'Your donation is much appreciated and it encourages us to continue to improve our content.',
+  rewardsBannerMonthlyText1: 'You can support this content creator by sending a tip. It\'s a way of thank them for making great content. Verified creators get paid for their tips during the first week of each calendar month.',
```
Comment (NejcZdovc) [inline .storybook/locale.ts]: where is this used?

**Pull request merged**